utils/tests_fetcher.py

```diff
@@ -492,12 +518,4 @@ def infer_tests_to_run(output_file, diff_with_last_commit=False, filters=None):
             print("Master branch detected, fetching tests against last commit.")
             diff_with_last_commit = True
 
-        try:
-            infer_tests_to_run(args.output_file, diff_with_last_commit=diff_with_last_commit, filters=args.filters)
-        except Exception as e:
-            print(f"\nError when trying to grab the relevant tests: {e}\n\nRunning all tests.")
-            with open(args.output_file, "w", encoding="utf-8") as f:
-                if args.filters is None:
-                    f.write("./tests/")
-                else:
-                    f.write(" ".join(args.filters))
```

Comment:
Why is this logic removed? And why are tests fetched not saved anymore?

Reply:
My bad, this was only for testing. Thank you for your careful review.